src/modules/lifecycle.js

```diff
@@ -20,8 +20,32 @@ const STOP_HOOK_FILTER = HOOK_NAME_FILTER.bind(undefined, 'stop');
 /**
  * Utilities for monitoring and updating the lifecycle of a PureCloud Client App
  *
+ * ### Lifecycle Hooks
+ *
+ * These utilities require the app to be opted into the appropriate lifecycle hook via
+ * advanced configuration.  This can be set via the API, Admin UI, or hard-coded for Premium Apps.
+ * The format of lifecycle hooks in advanced configuration is as follows:
+ *
+ * ```json
+ * {
+ *   "lifecycle": {
+ *     "ephemeral": <boolean>,
+ *     "hooks": {
+ *       "stop": <boolean>,
+ *       "blur": <boolean>,
+ *       "focus": <boolean>,
+ *       "bootstrap": <boolean>
+ *     }
+ *   }
+ * }
+ * ```
+ *
+ * See [Advanced Application Concepts - Lifecycle Events](https://developer.mypurecloud.com/api/client-apps/advanced.html#lifecycle_events) for more details.
```

Comment:
Use the path only so it works in all regions: `[Advanced Application Concepts - Lifecycle Events](/api/client-apps/advanced.html#lifecycle_events)`. Same for the rest of the URLs.

Reply:
I just realized this is a javascript file, not markdown docs for the dev center. A full URL may be more appropriate here in that case.

Reply:
It gets piped through jsdoc2md so it will end up as markdown after the build.

Relative links should work at that point, right?  If so, thanks for the suggestion.  Much better.

Reply:
Updated ... thanks for the catch